### DISCUSSION TITLE
How can I add a square box around a navbar icon, similar to the FumaDocs site?

### DISCUSSION THREAD

**Discussion by kakarj** [fuma-nama/fumadocs#1076] [Q&A]
<img width="262" alt="Screenshot 2024-11-20 at 8 02 56 AM" src="https://github.com/user-attachments/assets/4615dc89-7490-474d-ad75-92cabd3554d7">

I want it similar to the FumaDocs site.
<img width="262" alt="Screenshot 2024-11-20 at 7 57 50 AM" src="https://github.com/user-attachments/assets/7e11bd9d-f449-40ea-8307-1ed5dfda6683">

**Comment by fuma-nama** [comment]
In your loader’s icon option, wrap the icon in a div

**Reply by kakarj** [reply]
I wanted it to look like the FumaDocs design. I copied the FumaDocs code, but it's giving me an error in production:

icon.tsx:
<img width="1168" alt="Screenshot 2024-11-20 at 8 50 00 PM" src="https://github.com/user-attachments/assets/21459fba-9ef2-421a-a7e6-7017567102a6">

source.ts:
<img width="636" alt="Screenshot 2024-11-20 at 8 50 30 PM" src="https://github.com/user-attachments/assets/61bc7278-c792-423c-998a-3f9f8850bd21">

cn.ts:
export { twMerge as cn } from 'tailwind-merge';

error:
<img width="1217" alt="Screenshot 2024-11-20 at 8 56 48 PM" src="https://github.com/user-attachments/assets/31840f68-e154-44fd-a4cb-214b31896bc8">

**Reply by fuma-nama** [reply]
it's probably other things that caused the error instead of `icon` handler in `loader`, mind to provide a reproduction via GitHub or CodeSandbox?

**Comment by Gitstar-OC** [comment]
Is it solved do you need help ?

**Comment by Gitstar-OC** [comment]
![image](https://github.com/user-attachments/assets/777b723b-a1e8-428c-96bd-9f3f0d703479)

Do it like this below is the  code for it and other variations which are used on my website https://mindect.vercel.app/learn/sl 

```
import { CircuitBoard, BrainCircuit, BrainCog, Sigma, X, ChartColumn } from "lucide-react";
import { cn } from "@/lib/utils"

export function ProbabilityIcon({className}) { 
  return (
    <div className={cn(`bg-gradient-to-b from-[#FF6200] to-[#0F0C05] h-8 w-8 rounded-sm` , className)}>
      <ChartColumn className={cn(`text-[#FFC982] w-full  h-[90%] my-[5%]` , className)} />
    </div>
  );
}
```

And then use it in `docs/layout.tsx` as given below 
```
<RootToggle
              options={[
                {
                  title: "Algs",
                  description: "Learning Algorithms",
                  url: "/learn/algs",
                  icon: <AlgsIcon className="m-0" />,
                }]} after importing the icons 
```

**Reply by Gitstar-OC** [reply]
Don't try to copy all the things, most of the time it throws an error due to things like package versions and compiler.

**Reply by kakarj** [reply]
Thanks for your help! I'm still facing the same error. I'm working on an open source non-profit ai + education project similar to Mindect (I liked your work there). Can you help me fix this? If interested join our Discord: https://discord.gg/snCZb2fc :)

**Comment by Gitstar-OC** [comment]
Hope that is solved :)
